app/Http/Controllers/UserController.php

```diff
@@ -32,15 +36,58 @@ public function store(UserRequest $request)
     {   
         $authentication = Authentication::where('token', $request->authentication_token)
             ->where('status', AuthenticationStatus::MAIL_SENT)
-            ->where('expired_at', '>', Carbon::now())
+            ->where('expired_at', '>', now())
             ->first();
 
         if(is_null($authentication)){
             return to_route('tasks.index')->withErrors(['status_error' => '会員登録に失敗しました。']);
         }
+
+        $image = new Imagick();
+        $image->readImage($request->file('image_file'));
+        $archive_mimetype = config('mimetypes')[$image->getImageMimetype()];
```

Comment:
```suggestion
        $archive_extension = config('mimetypes')[$image->getImageMimetype()];
```
configの連想配列から取得している物は拡張子なので、変数名も適切な物にしてください。